.gitignore

```diff
@@ -59,3 +59,6 @@ typings/
 
 # next.js build output
 .next
+
+# test-results needed for CI but not needed on remote
```

Comment:
This comment confuses me, and I don't think it's necessary; I think it's going to be apparent to a reader why a folder named "test-results" should be .gitignored :)